exporters/dynamicconfig/notifieroptions.go

```diff
@@ -14,19 +14,43 @@
 
 package dynamicconfig
 
+import "go.opentelemetry.io/otel/sdk/resource"
+
 // Option is the interface that applies the value to a configuration option.
 type Option interface {
 	// Apply sets the Option value of a Config.
-	Apply(*ConfigNotifier)
+	Apply(*Notifier)
 }
 
-// WithConfigHost sets the ConfigHost configuration option of a Config.
+// WithConfigHost sets the configHost configuration option of a Notifier.
 func WithConfigHost(host string) Option {
 	return configHostOption(host)
 }
 
 type configHostOption string
 
-func (o configHostOption) Apply(notifier *ConfigNotifier) {
+func (o configHostOption) Apply(notifier *Notifier) {
 	notifier.configHost = string(o)
 }
+
+// WithProtoVersion sets the protoVersion configuration option of a Notifier
+func WithProtoVersion(version int32) Option {
```

Comment:
Just a heads up, I've gone ahead and deleted the `proto_version` field in the latest version of the protobuf